src/Components/SavedPooches/SavedPooches.js

```diff
@@ -29,4 +30,9 @@ const SavedPooches = ({savedDogs, unSaveDogPhoto}) => {
 }
 
 
-export default SavedPooches;
\ No newline at end of file
+export default SavedPooches;
+
+SavedPooches.propTypes = {
+  savedDogs: PropTypes.array.isRequired,
```

Comment:
So I'm not sure of what these props are specifically - this might be completely off-base as a result - but if I was to pick a small thing it might be defining more specific shapes for these incoming arrays.

Reply:
I think that's a great idea - this is in fact an array of objects so it'd be good to have more specificity on that.